website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -14,6 +14,57 @@ You can create a Kubeflow Pipeline built entirely using SageMaker components, or
 
 There are two versions of SageMaker components - boto3 based v1 components and SageMaker Operator for K8s (ACK) based v2 components. 
 
+
+## Configure Permissions
+
+Configure RBAC permissions for the service account used by kubeflow pipeline pods in the user/profile namespace. The pipeline runs are executed in user namespaces using the default-editor Kubernetes service account.
+
+
+Set the environment variable value for PROFILE_NAMESPACE(e.g. kubeflow-user-example-com) according to your profile and SERVICE_ACCOUNT name according to your installation:
+
+```
+export PROFILE_NAMESPACE=kubeflow-user-example-com
+export KUBEFLOW_PIPELINE_POD_SERVICE_ACCOUNT=default-editor
+```
+
+Create a RoleBinding that grants service account access to manage sagemaker custom resources
+
+```
+cat > manage_sagemaker_cr.yaml <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: manage-sagemaker-cr
+  namespace: ${PROFILE_NAMESPACE}
+subjects:
+- kind: ServiceAccount
+  name: ${KUBEFLOW_PIPELINE_POD_SERVICE_ACCOUNT}
+  namespace: ${PROFILE_NAMESPACE}
+roleRef:
+  kind: ClusterRole
+  name: ack-sagemaker-controller
+  apiGroup: rbac.authorization.k8s.io
+EOF
+
+kubectl apply -f manage_sagemaker_cr.yaml
+```
+Check rolebinding was created by running kubectl get rolebinding
+
+```
+manage-sagemaker-cr -n ${PROFILE_NAMESPACE} -oyaml
+```
+
+(Optional) If you are also using the SageMaker components version 1. Grant SageMaker access to the service account used by kubeflow pipeline pods.
```

Comment:
Add a section - Configuration for SageMaker components version 1

and talk about optional part in text after explaining what user is doing in this section instead of making the section optional (people tend to skip optional things without completely reading)

Also mention this step need to be done in each profile namespace